pyeudiw/tests/x509/test_x509.py

```diff
@@ -90,29 +93,36 @@ def gen_chain() -> list[bytes]:
         .subject_name(
             x509.Name(
                 [
-                    x509.NameAttribute(NameOID.COMMON_NAME, "leaf.example.org"),
+                    x509.NameAttribute(NameOID.COMMON_NAME, leaf_cn),
                 ]
             )
         )
         .issuer_name(intermediate.subject)
         .public_key(leaf_private_key.public_key())
         .serial_number(x509.random_serial_number())
-        .not_valid_before(datetime.utcnow())
-        .not_valid_after(datetime.utcnow() + timedelta(days=365))
+        .not_valid_before(date - timedelta(days=1))
+        .not_valid_after(date + timedelta(days=365))
         .add_extension(
             x509.BasicConstraints(ca=False, path_length=None),
             critical=True,
         )
+        .add_extension(
+            x509.SubjectAlternativeName([
+                x509.DNSName(leaf_cn),
+                x509.UniformResourceIdentifier(f"https://{leaf_cn}/OpenID4VP")
```

Comment:
```suggestion
                x509.UniformResourceIdentifier(f"{default_client_id}")
```

the uri should bring the default client id, `OpenID4VP` is a variable, we should not stick it into the code.

Reply:
yeah, it was an example

*client_id would be good